spec/fixtures/pages.yml

```diff
@@ -43,11 +47,11 @@ fashion_collection:
 
 grid_layout_collection:
   slug: 'collections/grid_layout'
+  collection: grid_layout
   state: 1
   type: 'Page::Landing'
 
 draft_landing_page:
```

Comment:
it's being used in the ability specs, I'll assign a collection to it and keep it.
The only reason i didn't was because I was reusing it for the 'has_landing_page? when there is NO landing page' spec, but i'll use something else there.